cargo-process.el

```diff
@@ -574,7 +574,12 @@ Cargo: Build and execute with --example <name>."
 With the prefix argument, modify the command's invocation.
 SEARCH-TERM is used as the search term for the Cargo registry.
 Cargo: Search registry for crates."
-  (interactive "sSearch: ")
+  (interactive
+   (list (let* ((default-search-term (and (thing-at-point-looking-at "[[:alnum:]-_]+") (match-string-no-properties 0)))
```

Comment:
Can this be extracted to its own function with a descriptive name?

Reply:
fixed in ca0364c